### PR TITLE
Update UsingTheIOSubsystem.md

### PR DESCRIPTION
**Pull request by qinkunbao**

### CODE REVIEW

**Review by dthaler — APPROVED** [review]

File: docs/UsingTheIOSubsystem.md [hunk 1]
```diff
@@ -139,8 +139,8 @@ int create_alphabet_file(const char* path)
     if (!(stream = fopen(path, "w")))
         return -1;
 
-    /* Write the letters of the alphabet to the file. */
-    if (fwrite(alphabet, 1, sizeof(alphabet), stream) != sizeof(ALPHABET))
+    /* Write the letters of the ALPHABET to the file. */
```
Comment (dthaler) [inline docs/UsingTheIOSubsystem.md:142]:
```suggestion
    /* Write the letters of the alphabet to the file. */
```

**Comment by mingweishih** [comment]
Can you squash the comments? Thanks!

**Comment by qinkunbao** [comment]
@mingweishih Sure!

**Review by mingweishih — APPROVED** [review]
LGTM

**Pull request merged**